Stonx/Stonx/Controller/StocksViewController.swift

```diff
@@ -0,0 +1,406 @@
+        lineChartView.data = data
+    }
+    
+    // fake data
+    // sets the data entry for the data
+    let yvalue: [ChartDataEntry] = [
+        ChartDataEntry(x: 0.0, y: 10.0),
+        ChartDataEntry(x: 1.0, y: 5.0),
+        ChartDataEntry(x: 2.0, y: 7.0),
+        ChartDataEntry(x: 3.0, y: 5.0),
+        ChartDataEntry(x: 4.0, y: 10.0),
+        ChartDataEntry(x: 5.0, y: 6.0),
+        ChartDataEntry(x: 6.0, y: 5.0),
+        ChartDataEntry(x: 7.0, y: 7.0),
+        ChartDataEntry(x: 8.0, y: 8.0),
+        ChartDataEntry(x: 9.0, y: 12.0),
+        ChartDataEntry(x: 10.0, y: 13.0),
+        ChartDataEntry(x: 11.0, y: 5.0),
+        ChartDataEntry(x: 12.0, y: 7.0),
+        ChartDataEntry(x: 13.0, y: 3.0),
+        ChartDataEntry(x: 14.0, y: 15.0),
+        ChartDataEntry(x: 15.0, y: 6.0),
+        ChartDataEntry(x: 16.0, y: 6.0),
+        ChartDataEntry(x: 17.0, y: 7.0),
+        ChartDataEntry(x: 18.0, y: 3.0),
+        ChartDataEntry(x: 19.0, y: 10.0),
+        ChartDataEntry(x: 20.0, y: 12.0),
+        ChartDataEntry(x: 21.0, y: 15.0),
+        ChartDataEntry(x: 22.0, y: 17.0),
+        ChartDataEntry(x: 23.0, y: 15.0),
+        ChartDataEntry(x: 24.0, y: 10.0),
+        ChartDataEntry(x: 25.0, y: 10.0),
+        ChartDataEntry(x: 26.0, y: 10.0),
+        ChartDataEntry(x: 27.0, y: 17.0),
+        ChartDataEntry(x: 28.0, y: 13.0),
+        ChartDataEntry(x: 29.0, y: 20.0),
+        ChartDataEntry(x: 30.0, y: 24.0),
+        ChartDataEntry(x: 31.0, y: 25.0),
+        ChartDataEntry(x: 32.0, y: 27.0),
+        ChartDataEntry(x: 33.0, y: 25.0),
+        ChartDataEntry(x: 34.0, y: 30.0),
+        ChartDataEntry(x: 35.0, y: 55.0),
+        ChartDataEntry(x: 36.0, y: 58.0),
+        ChartDataEntry(x: 37.0, y: 40.0),
+        ChartDataEntry(x: 38.0, y: 43.0),
+        ChartDataEntry(x: 39.0, y: 53.0),
+        ChartDataEntry(x: 40.9, y: 55.0)
+    ]
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        navigationItem.title = "AAPL"
+        configureSubviews()
+        setupConstraints()
+        setData()
+    }
+
+    private func configureSubviews() {
+        view.addSubview(scrollView)
+        scrollView.addSubview(stackView)
+        scrollView.addSubview(tradeButtonBottom)
+        stackView.addArrangedSubview(tickerSymbol)
+        stackView.addArrangedSubview(stockPriceHStackView)
+        stackView.addArrangedSubview(lineChartView)
+        stackView.addArrangedSubview(aboutLabelHeader)
+        stackView.addArrangedSubview(aboutTextLabel)
+        stackView.addArrangedSubview(typeHeaderLabel)
+        stackView.addArrangedSubview(typeTextLabel)
+        stackView.addArrangedSubview(marketStatsHeaderLabel)
+        stackView.addArrangedSubview(marketCapHStackView)
+        stackView.addArrangedSubview(stockVolumeHStackView)
+        stackView.addArrangedSubview(stockPERatioHStackView)
+        stackView.addArrangedSubview(stockEPSHStackView)
+    }
+
+    private func setupConstraints() {
```

Comment:
I created an extension for these that you can probably use to make alll of these one line

Reply:
tutorial:
<img width="1123" alt="Screen Shot 2022-10-31 at 5 07 07 PM" src="https://user-images.githubusercontent.com/29695936/199131720-08607c35-a755-4329-8a76-59a9447d1ddd.png">